tests/lib/account.js

```diff
@@ -234,6 +234,7 @@ define([
               keys: true,
               metricsContext: {
                 deviceId: '0123456789abcdef0123456789abcdef',
+                entrypoint: 'mock-entrypoint',
```

Comment:
It looks like `entrypoint` isn't required and isn't tested for here, is this change just for hygiene?

Reply:
No, my thinking was more that it asserts adding `entrypoint` doesn't cause a 400 error when testing against a local server.